internal/era/verify.go

```diff
@@ -0,0 +1,64 @@
+package era
+
+// func (r *Reader) Verify() error {
```

Comment:
Looks like this code was moved to era/main and is not needed anymore

Reply:
done